src/components/Checkbox/index.tsx

```diff
@@ -116,4 +138,3 @@ export default class Checkbox extends Component {
     );
   }
 }
```

Comment:
This should fix the [broken demo page](https://deploy-preview-257--cauldron-react.netlify.com/components/Checkbox)
```suggestion
}
Checkbox.displayName = 'Checkbox'
```

Reply:
Added this as a static display value instead